### PR TITLE
fix: 🐛 update fabric sdk to v2.2.8

### PR DESCRIPTION
**Pull request by ccordi**
This PR updates the fabric sdk to use 2.2.8.

This, along with the updated @grpc/grpc-js package, resolves a gRPC connection error when fabriclogger is started prior to channel creation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,8 +56,8 @@
     "asn1js": "^2.0.26",
     "debug": "^4.1.1",
     "express": "^4.17.1",
-    "fabric-common": "2.2.5",
-    "fabric-network": "^2.2.5",
+    "fabric-network": "^2.2.8",
+    "fabric-common": "^2.2.8",
```
Comment (atoulme) [inline package.json:60]: why this change to alphabetical order?

Comment (ccordi) [reply package.json:59]: yeah, that was unintentional; reverted.

**Review by atoulme — APPROVED** [review]
